pandas/core/series.py

```diff
@@ -938,6 +938,11 @@ def view(self, dtype: Dtype | None = None) -> Series:
         4      2
         dtype: int8
         """
+        warnings.warn(
+            "Series.view is deprecated and will be removed in a future version.",
```

Comment:
Point to astype instead if the goal was to change the dtype?

Reply:
Yeah